backend/audit/views/home.py

```diff
@@ -20,3 +20,16 @@ def get(self, request, *args, **kwargs):
         template_name = "home.html"
         extra_context = {"DISABLE_AUTH": settings.DISABLE_AUTH}
         return render(request, template_name, extra_context)
+
+
+class Maintenance(generic.View):
+    """
+    This is the redirected path for Maintenance mode.
+
+    It will return the home template with an error status for every single request
+    so long as maintenance is enabled.
+    """
+
+    def get(self, request, *args, **kwargs):
+        template_name = "500.html"
+        return render(request, template_name)
```

Comment:
I recommend we use a [503](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) here. So for our purposes, just a new HTML file much like the others. Eventually - it's not required now